addons/mail/models/mail_thread.py

```diff
@@ -2617,6 +2611,14 @@ def _notify_classify_recipients(self, recipient_data, model_name, msg_vals=None)
             for group_name, group_func, group_data in groups:
                 if group_func(recipient):
                     group_data['recipients'].append(recipient['id'])
+                    group_button_access = group_data['button_access']
+                    context = {'lang': recipient['lang']} if 'lang' in recipient else {}
```

Comment:
I don't get it, it seems like we never use `context`? Shouldn't you be using `with_context(**context):`?

Reply:
I think we would likely just create a task in the RD Fun SM project

Reply:
@hamzaislam101 oh, sorry forgot to respond.

He should be back next week

Reply:
Hey, I did forget a bit about this sorry.
Was waiting to check with @tde-banana-odoo 

Do we want to merge this as a hotfix for 15.0? <- Not a fan because we translate for a whole group based on a single recipient
Do we create a task on master to make this possible, adding language-specific groupings? <- Seems to make sense

Reply:
@hamzaislam101 
Well, after looking a bit into it, if we're fine with a fix in master `https://github.com/odoo/odoo/pull/121498` should be taking care of it as we already classify by language on master

You can follow https://www.odoo.com/web#id=3370576&cids=1&menu_id=4720&action=333&active_id=965&model=project.task&view_type=form to see when that's merged.

I'll try poking that task see if we can make it move :)